### PR TITLE
Blog app/controllers

### PR DESCRIPTION
**Pull request by AmaliaGM**
## Project requirements

- Important: do NOT use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
- Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:
All users page wireframe Single user page wireframe All posts by a given user page wireframe Single post page wireframe

- For each URL you should create:
A route.
An action in the correct controller.
A view file.

- View files should include only placeholders written in plain HTML

- Make sure that you added code to handle only the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'app#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :app
```
Comment (descholar-ceo) [inline config/routes.rb]: - [ ] Kindly make sure that you have the following routes:
```  
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
```

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @AmaliaGM 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] The PR is following the correct git / github flow ✔️ 
- [ ] The PR has linters installed and with no errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] The `users_controller` is missing
- [ ] The `post_controller` is missing
- [ ] The `user` views related to users controller are missing
- [ ] The `post` views related to post controllers are missing

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/app/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<h1>New Post</h1>
+
+<%= form_with model: @post do |form| %>
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
```
Comment (Davidosky007) [inline app/views/app/new.html.erb]: * As requested by the previous reviewer The user views related to the users' controller are missing and
 The post views related to post controllers are missing, Kindly add view files for each of the controller(posts, users) actions(index, show) [For refresher check project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=For%20each%20URL,the%20following%20output%3A)

File: config/routes.rb (hunk 1 above)
Comment (Davidosky007) [reply config/routes.rb]: * This issue hasn't been resolved, You need to create routes for users and posts 
* you can use write your routes like this 👇 
 ```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
```
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @AmaliaGM 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Well written PR description 👍 
* No linter errors 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @AmaliaGM 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**